### PR TITLE
Added get_fileusage

### PR DESCRIPTION
**Pull request by ayuhsya**
Added method to fetch `Fileusage` in `wikifile.py`, takes `filenames` as input.

### CODE REVIEW

File: sample/wikifile.py [hunk 1]
```diff
@@ -59,8 +59,40 @@ def get_duplicatefiles(self, dflimit="max", dfdir="ascending", dflocalonly=False
         self.payload['prop'] = None
         return df_list
 
+    def get_fileusage(self, fuprop="pageid|title|redirect", fulimit="max", funamespace="*", fushow="!redirect"):
```
Comment (abinashmeher999) [inline sample/wikifile.py:62]: `fuprop` isn't being used anywhere. What is its purpose?

**Review by abinashmeher999 — CHANGES_REQUESTED** [review]

**Comment by abinashmeher999** [comment]
Everything else looks great :+1:

File: sample/wikifile.py (hunk 1 above)
Comment (ayuhsya) [reply sample/wikifile.py:62]: Oops.

Comment (abinashmeher999) [reply sample/wikifile.py:62]: How is it used?

File: sample/wikifile.py [hunk 2]
```diff
@@ -59,8 +59,41 @@ def get_duplicatefiles(self, dflimit="max", dfdir="ascending", dflocalonly=False
...
+            self.payload['fucontinue'] = res['continue']['fucontinue']
+            res = requests.get(self.base_url, params=self.payload, headers=self.headers).json()
+            for page_id, page_content in res['query']['pages'].items():
+                if prop not in page_content:
+                    continue
+                if page_content['title'] not in fu_list:
+                    fu_list[page_content['title']] = []
+                fu_list[page_content['title']] += page_content[prop]
+
+        self.payload.pop('fucontinue', None)
```
Comment (abinashmeher999) [inline sample/wikifile.py:88]: You will have to pop `fuprop` too.

**Review by abinashmeher999 — CHANGES_REQUESTED** [review]

**Comment by abinashmeher999** [comment]
What happened?

**Comment by ayuhsya** [comment]
TIFU, created a new PR.

**Pull request closed without merging**